packages/xo-web/src/common/intl/messages.js

```diff
@@ -1230,8 +1230,16 @@ const messages = {
   managementAgentNotDetected: 'Management agent not detected',
   noIpv4Record: 'No IPv4 record',
   noIpRecord: 'No IP record',
+  secureBootEnforced: 'Secure boot enforced',
+  secureBootEnforcedPendingBoot: 'Secure boot enforced, pending first boot',
```

Comment:
If it's not too late, I realize the following wording would be better:
```suggestion
  secureBootWantedPendingBoot: 'Secure boot wanted, pending first boot',
```